apps/src/studio/levels.js

```diff
@@ -1125,7 +1125,8 @@ levels.iceage_8 = utils.extend(levels.playlab_8, {
 });
 levels.gumball_8 = utils.extend(levels.playlab_8, {
   background: 'wood',
-  avatarList: ['bananajoe', 'antony']
+  avatarList: ['bananajoe', 'antony'],
+  'delayCompletion': 500,
```

Comment:
as mentioned, better sprite collision rectangles might be the better solution here, but this can make things look better for now.